assets/src/edit-story/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -139,12 +140,26 @@ function TextSetsPane({ paneRef }) {
     [textSets]
   );
 
-  const handleSelectedCategory = useCallback((selectedCategory) => {
-    setSelectedCat(selectedCategory);
-    localStore.setItemByKey(`${LOCAL_STORAGE_PREFIX.TEXT_SET_SETTINGS}`, {
-      selectedCategory,
-    });
-  }, []);
+  const speak = useLiveRegion('polite');
```

Comment:
nit:`'polite'` is the default so this isn't needed but I don't think it's a big deal so it's fine if we leave it too! Love seeing this used more though